src/Buffer.mo

```diff
@@ -53,7 +60,7 @@ module {
       1;
```

Comment:
curious, I wonder why the formatter didn't delete that semi after 1; but did delete the last semi in the else  branch....

(Nothing to fix here)

Reply:
@rvanasa

Reply:
This works as expected in the latest formatter version (always a trailing semicolon for multi-line blocks). Maybe it wasn't originally formatted? Otherwise, definitely let me know and I'll look into it.